### PR TITLE
[office-js-preview] (Outlook) Document the JS runtime URL property

### PR DESCRIPTION
**Pull request by samantharamon**
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: None.
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

**Review by Adrian-MSFT — APPROVED** [review]

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -1006,6 +1006,20 @@ declare namespace Office {
          * Provides objects and methods that you can use to create and manipulate UI components, such as dialog boxes.
          */
         ui: UI;
+        /**
+         * Gets the object to retrieve the runtime URL of an add-in.
```
Comment (AlexJerabek) [inline types/office-js-preview/index.d.ts:1010]: The description reads as if it's for a singular URL, but the property is plural. Seems inconsistent.

File: types/office-js-preview/index.d.ts [hunk 2]
```diff
@@ -1443,6 +1457,51 @@ declare namespace Office {
          */
         openBrowserWindow(url: string): void;
     }
+    /**
+     * Provides the runtime URL of an add-in.
```
Comment (AlexJerabek) [inline types/office-js-preview/index.d.ts:1461]: Relatedly, if the plan is to make this API ready for future upgrades, consider leaving the class-level description general enough to accommodate that.

File: types/office-js-preview/index.d.ts (hunk 1 above)
Comment (samantharamon) [reply types/office-js-preview/index.d.ts:1010]: Fixed.

File: types/office-js-preview/index.d.ts (hunk 2 above)
Comment (samantharamon) [reply types/office-js-preview/index.d.ts:1461]: Fixed.

**Review by AlexJerabek — APPROVED** [review]

**Review by RyanCavanaugh — APPROVED** [review]

**Comment by samantharamon** [comment]
Ready to merge

**Pull request merged**